src/main/java/de/tum/in/www1/artemis/config/SecurityConfiguration.java

```diff
@@ -129,6 +129,7 @@ protected void configure(HttpSecurity http) throws Exception {
             .antMatchers("/api/account/reset-password/finish").permitAll()
             .antMatchers("/api/lti/launch/*").permitAll()
             .antMatchers("/api/files/attachments/**").permitAll()
+            .antMatchers("/api/files/file-upload-submission/**").permitAll()
```

Comment:
after the last changes we do not need this anymore, right?

Reply:
Yes this is still needed for file download